### PR TITLE
news: darktable 3.4.0 released.

### PR DESCRIPTION
**Pull request by TurboGit**
The announce for the 3.4 release, to be published on 24th morning.

### CODE REVIEW

File: content/news/2020-12-24-darktable-3.4.0-released/2020-12-24-darktable-3.4.0-released.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+  to write text for contributors. Feedback or contributions for the
+  new format are welcome at https://github.com/darktable-org/dtdocs.
+
+  Current manual (multilingual):
+
+  https://darktable.gitlab.io/doc/
+
+  New manual (English only):
+
+  https://darktable-org.github.io/dtdocs/
```
Comment (elstoc) [inline content/news/2020-12-24-darktable-3.4.0-released/2020-12-24-darktable-3.4.0-released.md:53]: This has now been published to https://www.darktable.org/usermanual/en/ in preparation for 3.4

Comment (TurboGit) [reply content/news/2020-12-24-darktable-3.4.0-released/2020-12-24-darktable-3.4.0-released.md:53]: Fixed, thanks!

**Comment by elstoc** [comment]
@TurboGit while you're there please could you include a link to the dtdocs pages in the resources section as well.

**Comment by TurboGit** [comment]
@elstoc : Done.

**Pull request merged**